### PR TITLE
Added HPC SIG champions post

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 1]
```diff
@@ -40,31 +40,30 @@ is due to shut down this year and will be replaced with the [ARCHER 2][archer2-n
...
 between national services (e.g. Archer, Tier-1) and local university HPCs (Tier-3). They are also testbeds for new and 
 heterogeneous HPC technologies, many geared to solve particular type of algorithm or scientific problem.
 
 If you're in an institution that's associated with these systems and are doing research inline with 
 the stated purpose of system then access is often free at point-of-use so get in touch with your local Tier-2 contact 
-to get more (and free) compute for your research! For example, if you're doing Deep Learning or Molecular Dynamics simulation here at Sheffield you can 
-request JADE access by filling in this [JADE access form][jade-access-form]. We're also associated with NICE but the system
-has not come on-line yet.
+to get more (and free) compute for your research! For example, **if you're doing Deep Learning or Molecular Dynamics simulation here at Sheffield you can 
+request JADE access by filling in this [JADE access form][jade-access-form]**. We're also associated with NICE and will be able to provide access once the comes on-line.
```
Comment (ptheywood) [inline _posts/2020-02-28-hpc-sig-champions.md]: > once the comes on-line

Missing a word?

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+planned and prolonged system maintenance, relocations, etc. The cloud can provide an extra resiliency by allowing on-demand clones of
+nodes to be brought on-line relatively quickly in times of failure or to provide extra compute capacity. 
+
+Liverpool has been trialing cloud bursting on Amazon's AWS through Alces Flight. Their setup uses local gateway/appliance 
+nodes to provide authentication and supply node images. Additionally, front-end login nodes and storage are hosted locally 
+and user's data are pushed to the cloud on-demand. Cloud storage was still found to be relatively expensive and cheaper archival storage (e.g. glacier)
+took too long to retrieve data when needed. Cliff also mentioned that the cheapest nodes may not be the best e.g. for AWS while t2.medium is cheapest, 
+c4.large and c5.large allows work to finish faster.
+
+Bath is till running a pilot project with Red Oak Consulting on Microsoft's Azure platform. Early issues encountered are unexpected
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: bath is 'till' should be still

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+and user's data are pushed to the cloud on-demand. Cloud storage was still found to be relatively expensive and cheaper archival storage (e.g. glacier)
+took too long to retrieve data when needed. Cliff also mentioned that the cheapest nodes may not be the best e.g. for AWS while t2.medium is cheapest, 
+c4.large and c5.large allows work to finish faster.
+
+Bath is till running a pilot project with Red Oak Consulting on Microsoft's Azure platform. Early issues encountered are unexpected 
+clauses in software agreements and delays in cost reporting and ability to map true cost to workloads. They expect the trial to be completed by summer.
+ 
+
+## ARCHER 2
+[ARCHER][archer], a UK's national supercomputing service for problems with very large compute requirements (e.g. climate modelling),
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: "a UK's", presumably should be "the UK's"

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+is due to shut down this year and will be replaced with the [ARCHER 2][archer2-news] system.
+
+![ARCHER 2 transition timeline](/assets/images/2020-02-28-hpc-sig-champions/archer-2-transition-timeline.jpg)
+
+Alan Simpson (EPCC) provided the transition timeline shown in the image above. It is expected that a 4-cabinet test system to be available by mid-june and the whole system to go
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: 'to be' -> 'will be'?

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+It's also interesting to see the various [training courses provided by the ARCHER team][archer-training], both class based and on-line. There's also the driving test which 
+awards a small amount of time on the service for completing the course. I've been told that all of these courses are available for all researchers 
+throughout the UK.
+
+
+## Tier-2 HPC Refresh
+Tier-2 HPCs are EPSRC funded regional-level supercomputing clusters, they are designed to bridge the computational power requirement gap 
+between national services (e.g. Archer, Tier-1) and local university HPCs (Tier-3). They are also testbeds for new and 
+heterogeneous HPC technologies, many geared to solve particular type of algorithm or scientific problem.
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: 'particular type' either 'a particular type' or 'particular types'

File: _posts/2020-02-28-hpc-sig-champions.md (hunk 1 above)
Comment (Robadob) [reply _posts/2020-02-28-hpc-sig-champions.md]: 'the' -> 'that' ?

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+
+If you're in an institution that's associated with these systems and are doing research inline with 
+the stated purpose of system then access is often free at point-of-use so get in touch with your local Tier-2 contact 
+to get more (and free) compute for your research! For example, **if you're doing Deep Learning or Molecular Dynamics simulation here at Sheffield you can 
+request JADE access by filling in this [JADE access form][jade-access-form]**. We're also associated with NICE and will be able to provide access once the comes on-line.
+
+Even if you're not associated with one of the Tier-2 institution, it is still possible to apply for access through the [EPSRC Resource Allocation Panel (RAP)][tier2-rap]. 
+Additionally, the [hpc-uk][hpc-acuk] website offer other ways of accessing the systems and has useful links to HPC training courses. 
+
+Funding has been approved for refresh of the Tier-2 centres since late last year and for this round we have an addition of
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: 'for refresh' -> 'for a refresh'/'for the refresh'

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+## Wrap-up Discussions
+HPC Champions wrapped up with discussions on various Tier-2 and general HPC related topics. These include key performance indicators, 
+whether it should be judged by the metric of number of papers generated or impact of papers. Training and on-boarding of HPC users, how there
+should be a form of standardisation of course materials/curriculum between the different centres and how they can collaborate together to generate this.
+The silo of technical instructions and issues behind private ticketing systems and how this can be opened up, possibly through sharing on 
+open platforms such as stack exchange. Packaging of HPC software, how the users can be allowed flexibility to install their own software using
+package managers such as conda, Spack or EasyBuild and how the centres should also give support in building software optimised for their own cluster.
+
+## Conclusion 
+It's definitely an eventful year for UK HPC with all of the refresh happening. It's great to see the variety of CPU platforms, with just about half
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: refreshes?

File: _posts/2020-02-28-hpc-sig-champions.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+open platforms such as stack exchange. Packaging of HPC software, how the users can be allowed flexibility to install their own software using
+package managers such as conda, Spack or EasyBuild and how the centres should also give support in building software optimised for their own cluster.
+
+## Conclusion 
+It's definitely an eventful year for UK HPC with all of the refresh happening. It's great to see the variety of CPU platforms, with just about half 
+of the centres opting for AMD, ARM or POWER chipsets as their main offering. There's also a big investment in GPU enabled nodes as Deep Learning is
+being adopted into more research workflows.  
+
+That's all from the HPC-SIG and HPC Champions this time, thank you for reading. If you have HPC problems you're struggling with, reach out to 
+the hpc@sheffiled.ac.uk google groups. We at the [RSE][rse] also run fortnightly [Code Clinic][code-clinic] sessions to help with HPC or other general technical problems in your research.
```
Comment (Robadob) [inline _posts/2020-02-28-hpc-sig-champions.md]: group singular?

**Review by Robadob — COMMENTED** [review]
Lots of small word changes, can't speak for the content.

File: _posts/2020-02-28-hpc-sig-champions.md (hunk 1 above)
Comment (twinkarma) [reply _posts/2020-02-28-hpc-sig-champions.md]: Thanks for commenting. Fixed in next commit.

File: _posts/2020-02-28-hpc-sig-champions.md (hunk 2 above)
Comment (twinkarma) [reply _posts/2020-02-28-hpc-sig-champions.md]: Thanks for commenting. Fixed in next commit along with your other corrections.

**Pull request merged**